src/hooks/useStore.js

```diff
@@ -0,0 +1,35 @@
+import create from 'zustand';
+import {persist} from 'zustand/middleware';
+
+const shoppingItemsStore = set => ({
+  shoppingItems: [],
+  addShoppingItem: item => {
+    set(state => ({
+      shoppingItems: [item, ...state.shoppingItems],
+    }));
+  },
+  removeShoppingItem: itemId => {
+    set(state => ({
+      shoppingItems: state.shoppingItems.filter(item => item.id !== itemId),
+    }));
+  },
+
+  plusItemQuantity: itemId => {
+    set(state => ({
+      shoppingItems: state.shoppingItems.map(item =>
+        item.id === itemId ? {...item, quantity: item.quantity + 1} : item
+      ),
+    }));
+  },
+  minusItemQuantity: itemId => {
```

Comment:
I think nicer names for these functions would be `increaseItemQuantity` and `decreaseItemQuantity`. That way it would be a bit nicer to read and understand that these are functions :)
```suggestion
  increaseItemQuantity: itemId => {
    set(state => ({
      shoppingItems: state.shoppingItems.map(item =>
        item.id === itemId ? {...item, quantity: item.quantity + 1} : item
      ),
    }));
  },
  decreaseItemQuantity: itemId => {
```

Reply:
Yeah, that's right.